misc/review/MidiServiceTransportPluginInfo.idl

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+// ============================================================================
+// This is part of the Windows MIDI Services App API and should be used
+// in your Windows application via an official binary distribution.
+// Further information: https://github.com/microsoft/MIDI/
+// ============================================================================
+
+#include "namespace_defines.h"
+#include "midl_defines.h"
+MIDI_IDL_IMPORT
+
+namespace MIDI_MIDL_NAMESPACE
+{
+    MIDI_API_CONTRACT_v1
+    [default_interface]
+    runtimeclass MidiServiceTransportPluginInfo
+    {
+        Guid Id{ get; };
+
+        String Name{ get; };
+        String Mnemonic{ get; };
```

Comment:
Mnemonic is usually for keyboard accessibility. Is this a single letter that the plugin wants to use as a shortcut?

Is this just "ShortName"? (E.g., Name = "Contoso Deluxe Crunch Plugin", Short name = "Crunch")

Reply:
It's a 2-4 letter abbreviation or acryonym, or random characters that we associate with the transport. It's more the classic non-technical definition of menmonic. But I'm open to other names here.

Right now, the values are

DIAG = Diagnostic
LOOP = Loopback
KS = Kernel Streaming
KSA = Kernel Streaming Aggregate
BLE1 = Bluetooth LE MIDI 1.0
APP = App-to-app MIDI (Virtual Device)

Those short values are also used in the Endpoint Device Id created. For example:

![image](https://github.com/microsoft/MIDI/assets/1421146/08db860c-577f-4c19-b2db-53358deca8f9)